scripts/install2

```diff
@@ -119,213 +187,43 @@ if [ -z $(echo $PATH | grep "/sbin")  ]; then
 fi
-if [ -z "$WITH_FIREWALL" -o "$WITH_FIREWALL" != "true" -a "$WITH_FIREWALL" != "false" ]; then
-    read -p "Configure Basic Firewall (Y/n)? " -n 1 -r
-    echo ""
-    if [[ $REPLY =~ ^[Nn]$ ]]; then
-        echo -e "\e[1;31mInternet Firewall will be skipped\e[0m"
-        WITH_FIREWALL=false
-    else
-        echo -e "\e[1;32mInternet Firewall will be configured\e[0m"
-        WITH_FIREWALL=true
-    fi
-fi
-if [ -z "$WITH_CJDNS_IPTUNNEL" -o "$WITH_CJDNS_IPTUNNEL" != "true" -a "$WITH_CJDNS_IPTUNNEL" != "false" ]; then
-    read -p "Configure Internet gateway (Y/n)? " -n 1 -r
-    echo ""
-    if [[ $REPLY =~ ^[Nn]$ ]]; then
-        echo -e "\e[1;31mInternet gateway configuration will be skipped\e[0m"
-        WITH_CJDNS_IPTUNNEL=false
-    else
-        echo -e "\e[1;32mInternet gateway will be configured\e[0m"
-        WITH_CJDNS_IPTUNNEL=true
-    fi
-fi
-if [ -z "$WITH_IPFS" -o "$WITH_IPFS" != "true" -a "$WITH_IPFS" != "false" ]; then
-    read -p "Install IPFS (Y/n)? " -n 1 -r
-    echo ""
-    if [[ $REPLY =~ ^[Nn]$ ]]; then
-        echo -e "\e[1;31mIPFS installation will be skipped\e[0m"
-        WITH_IPFS=false
-    else
-        echo -e "\e[1;32mIPFS will be installed\e[0m"
-        WITH_IPFS=true
-    fi
+if [ "$SUPPORT_HOSTAP" == "true" ]; then
+    askModule "WITH_WIFI_AP" "WiFi Access Point"
 fi
+askModule "WITH_FIREWALL" "Basic Firewall"
+askModule "WITH_CJDNS_IPTUNNEL" "Internet gateway"
+askModule "WITH_IPFS" "IPFS"
 if [ "$WITH_IPFS" == true -a "$BOARD_FAMILY" == "Raspberry Pi" ]; then
-    if [ -z "$WITH_IPFS_PI_STREAM" -o "$WITH_IPFS_PI_STREAM" != "true" -a "$WITH_IPFS_PI_STREAM" != "false" ]; then
-        read -p "Install IPFS Pi stream (y/N)? " -n 1 -r
-        echo ""
-        if [[ $REPLY =~ ^[Yy]$ ]]; then
-            echo -e "\e[1;32mIPFS Pi stream will be installed\e[0m"
-            WITH_IPFS_PI_STREAM=true
-        else
-            echo -e "\e[1;31mIPFS Pi stream installation will be skipped\e[0m"
-            WITH_IPFS_PI_STREAM=false
-        fi
-    fi
-fi
-if [ -z "$WITH_SSB" -o "$WITH_SSB" != "true" -a "$WITH_SSB" != "false" ]; then
-    read -p "Install SSB (Y/n)? " -n 1 -r
-    echo ""
-    if [[ $REPLY =~ ^[Nn]$ ]]; then
-        echo -e "\e[1;31mSSB installation will be skipped\e[0m"
-        WITH_SSB=false
-    else
-        echo -e "\e[1;32mSSB will be installed\e[0m"
-        WITH_SSB=true
-    fi
-fi
-if [ "$WITH_SSB" == true ]; then
-    if [ -z "$WITH_SSB_WEB" -o "$WITH_SSB_WEB" != "true" -a "$WITH_SSB_WEB" != "false" ]; then
-        read -p "Install SSB web client (Experimental) (Y/n)? " -n 1 -r
-        echo ""
-        if [[ $REPLY =~ ^[Yy]$ ]]; then
-            echo -e "\e[1;32mSSB web client will be installed\e[0m"
-            WITH_SSB_WEB=true        
-        else
-            echo -e "\e[1;31mSSB web client installation will be skipped\e[0m"
-            WITH_SSB_WEB=false
-                fi
-    fi
+    askModule "WITH_IPFS_PI_STREAM" "IPFS Pi stream" "n"
```

Comment:
This seems like a good time to standardize our capitalization strategy. The messages have capitalizations that are kind of all over the place.

Reply:
Decided to do Title Case (or whichever one it is that does not capitalize small words only)